{
	"id": "CVE-2021-28676",
	"description": "An issue was discovered in Pillow before 8.2.0. For FLI data, FliDecode did not properly check that the block advance was non-zero, potentially leading to an infinite loop on load.",
	"hardware": [],
	"software": [
		{
			"name": "pillow",
			"platform": "debian-12",
			"version": "< 8.1.2+dfsg-0.2"
		},
		{
			"name": "pillow",
			"platform": "debian-11",
			"version": "< 8.1.2+dfsg-0.2"
		},
		{
			"name": "pillow",
			"platform": "debian-10",
			"version": "*"
		},
		{
			"name": "pillow",
			"platform": "debian-9",
			"version": "< 4.0.0-4+deb9u3"
		},
		{
			"name": "pillow",
			"platform": "debian-12",
			"version": "< 8.1.2+dfsg-0.2"
		},
		{
			"name": "pillow",
			"platform": "debian-11",
			"version": "< 8.1.2+dfsg-0.2"
		},
		{
			"name": "pillow",
			"platform": "debian-10",
			"version": "*"
		},
		{
			"name": "pillow",
			"platform": "debian-9",
			"version": "< 4.0.0-4+deb9u3"
		},
		{
			"name": "pillow",
			"platform": "debian-12",
			"version": "< 8.1.2+dfsg-0.2"
		},
		{
			"name": "pillow",
			"platform": "debian-11",
			"version": "< 8.1.2+dfsg-0.2"
		},
		{
			"name": "pillow",
			"platform": "debian-10",
			"version": "*"
		},
		{
			"name": "pillow",
			"platform": "debian-12",
			"version": "< 8.1.2+dfsg-0.2"
		},
		{
			"name": "pillow",
			"platform": "debian-11",
			"version": "< 8.1.2+dfsg-0.2"
		},
		{
			"name": "pillow",
			"platform": "debian-10",
			"version": "*"
		}
	],
	"references": [
		"https://pillow.readthedocs.io/en/stable/releasenotes/8.2.0.html#cve-2021-28676-fix-fli-dos",
		"https://github.com/python-pillow/Pillow/pull/5377",
		"https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/MQHA5HAIBOYI3R6HDWCLAGFTIQP767FL/",
		"https://security.gentoo.org/glsa/202107-33",
		"https://lists.debian.org/debian-lts-announce/2021/07/msg00018.html"
	],
	"severity": null,
	"state": "published",
	"_is_edited": false
}